src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -85,10 +88,36 @@ public static class ListPackageCommand
                     Strings.NuGetXplatCommand_Interactive,
                     CommandOptionType.NoValue);
 
+                var verbosity = listpkg.Option(
+                    "-v|--verbosity",
+                    Strings.Verbosity_Description,
+                    CommandOptionType.SingleValue);
+
                 listpkg.OnExecute(async () =>
                 {
                     var logger = getLogger();
 
+                    switch (verbosity.Value()?.ToUpperInvariant())
```

Comment:
This feels weird. Do other dotnet tools embed verbosity strings? 
We do have NuGet.Common.LogLevel

Reply:
AFAIK this is the first place where such logic exists. It's a mapping from MSBuild verbosity to NuGet log level, so only NuGet code could possibly have this. The existing places in the code have the reverse mapping, such as https://github.com/NuGet/NuGet.Client/blob/05925a63fde32ab277ccaab13ed466add9ac9dc8/src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/NuGetSdkLogger.cs.

Please let me know if you'd like to see this in a different shape or place.

Reply:
I moved the code to a helper method per @nkolev92's feedback. I think it looks a little less out of place that way.